frontend/src/views/scrimmaging.js

```diff
@@ -225,6 +243,25 @@ class ScrimmageHistory extends Component {
                       ) : (
                         <td>N/A</td>
                       )}
+                      <td>
+                        {" "}
+                        <a
+                          style={{ cursor: "pointer" }}
+                          onClick={($event) => {
+                            // Prevent default behavior when clicking a link
+                            $event.preventDefault();
+
+                            let fileURL = URL.createObjectURL(
+                              new Blob([s.logs], { type: "text/plain" })
+                            );
+                            window.open(fileURL);
+                          }}
+                          target="_blank"
+                          rel="noopener noreferrer"
+                        >
+                          View log
+                        </a>{" "}
+                      </td>
```

Comment:
Might as well delete? See #436 -- seems better to add logs-exposure only once logs are sanitized or made private to admins (https://mit-battlecode.slack.com/archives/C03SJ354E2K/p1672807451869729)